test/cases/api.sh

```diff
@@ -264,6 +264,11 @@ popd
 # Install the necessary cloud provider client tools
 #
 installClient
+# install the govc client for the vmware tests which are run only for (generic) S3
+# the installClientVsphere is sourced by api/aws.s3.sh and api/generic.s3.sh
+if [ "${IMAGE_TYPE}" == "${IMAGE_TYPE_VSPHERE}" ]; then
+    installClientVSphere
+fi
```

Comment:
Feels a bit off here.

Could we rename `installClient` in common/aws.sh to `installAWSClient`; and then implement `installClient` in `aws.s3.sh` and `aws.sh`. And in the latter it just calles `installAWSClient`, and the former it also installs the vsphere client?

Reply:
yeah, that should be possible. I like it... 🙂

Reply:
I reworked this part based on your suggestion. It should be all much nicer now 😉